html/elements/link.json

```
@@ -499,6 +499,57 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "secure_context_only": {
```

It's not consistent, but `secure_context_required` is much more common for this sort of thing. Mind changing it?

```suggestion
          "secure_context_required": {
```